Tests/Controller/DefaultControllerTest.php

```diff
@@ -8,10 +8,8 @@ class DefaultControllerTest extends WebTestCase
 {
     public function testIndex()
     {
-        $client = static::createClient();
-
-        $crawler = $client->request('GET', '/');
-
-        $this->assertContains('Hello World', $client->getResponse()->getContent());
+        //$client = static::createClient();
+        //$crawler = $client->request('GET', '/');
+        //$this->assertContains('Hello World', $client->getResponse()->getContent());
```

Comment:
Ilyen esetben inkább töröld a fájlt, nem kellenek felesleges kódok a rendszerbe. Mivel ez az egyetlen fájl a könyvtárban, ezért a `Tests/Controller` könyvtár is törölhető.